presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -69,6 +63,38 @@
      */
     private boolean hideInternalColumns = true;
 
+    /**
+     * The Zookeeper URI.
+     * Example: hostAddress:2181
+     */
+    private String kafkaZookeeperUri = "sdzookeeper.smf1.twitter.com:2181";
+
+    /**
+     * The Zookeeper path for Kafka services.
+     * Example: /twitter/service/kafka/prod/tfe
+     */
+    private String kafkaServicePath = "/twitter/service/kafka/prod/tfe";
```

Comment:
What if we need to read other topic rather than tfe in the future?

Reply:
This is also configurable as above. All values in this file are read through config files. For particular topic we are interested in, we can pass in through parameter `kafka.zookeeper.path`

```
@Config("kafka.zookeeper.path")
    public KafkaConnectorConfig setZkPath(String zkPath)
    {
        this.kafkaServicePath = zkPath;
        return this;
    }
```